data/mods/Aftershock/items/comestibles/alienfood.json

```diff
@@ -132,5 +173,406 @@
     "vitamins": [ [ "mutant_toxin", 25 ] ],
     "stack_size": 1,
     "fun": -12
+  },
+  {
+    "id": "alien_flesh",
+    "copy-from": "flesh",
+    "type": "COMESTIBLE",
+    "name": { "str": "chunk of alien flesh", "str_pl": "chunks of alien flesh" },
+    "description": "Freshly butchered meat from an alien corpse.  You could eat it raw or cooked, but this is a completely alien biosphere.",
+    "weight": "296 g",
+    "volume": "250 ml",
+    "price": 500,
+    "price_postapoc": 50,
+    "spoils_in": "1 day",
+    "calories": 402,
+    "healthy": -1,
+    "fun": -10,
+    "cooks_like": "meat_cooked",
+    "vitamins": [ [ "vitC", 13 ], [ "calcium", 2 ], [ "iron", 53 ], [ "vitB", 778 ], [ "mutant_toxin", 25 ] ],
+    "flags": [ "SMOKABLE", "RAW", "PREDATOR_FUN" ],
+    "smoking_result": "meat_smoked"
+  },
+  {
+    "id": "alien_flesh_cooked",
+    "copy-from": "flesh",
+    "weight": "296 g",
+    "volume": "250 ml",
+    "spoils_in": "1 day",
+    "type": "COMESTIBLE",
+    "name": { "str_sp": "cooked alien flesh" },
+    "description": "This is a chunk of freshly cooked alien meat.  It's filling and nutritious, but you have worries.",
```

Comment:
```suggestion
    "description": "This is a chunk of freshly cooked alien meat.  It's could be either filling and nutritious, or poisonous and lethal.",
```

Reply:
```suggestion
    "description": "This is a chunk of freshly cooked alien meat.  It could be either filling and nutritious, or poisonous and perhaps lethal.",
```